docs/source/workflows/getting_started.rst

```diff
@@ -20,11 +20,12 @@ Design Workflow
 
 The :doc:`DesignWorkflow <design_workflows>` is the core AI workflow on the platform.
 This workflow generates scored candidates for Sequential Learning.
-To run, it requires one of each of the modules listed below and executes in the following manner:
+It executes in the following manner:
 
 -  Material candidates are pulled from the design space.
 -  The predictor adds additional information to the candidates.
--  Enriched candidates are scored, and the processor selects the next batch of candidates to pull from the design space.
+-  Enriched candidates are scored
+-  The Citrine Platform selects the next batch of candidates to pull from the design space.
```

Comment:
```suggestion
-  Enriched candidates are scored.
-  The Citrine Platform selects the next batch of candidates to pull from the design space.
```
Or maybe something else was meant?  This looks like half an edit.

Reply:
I just added a missing period

Reply:
Sorry, now it's fixed